packages/pangea-sdk/pangea/asyncio/services/audit.py

```diff
@@ -272,7 +272,7 @@ async def search(
         end: Optional[Union[datetime.datetime, str]] = None,
         limit: Optional[int] = None,
         max_results: Optional[int] = None,
-        search_restriction: Optional[dict] = None,
+        search_restriction: Optional[dict[str, Sequence[str]]] = None,
```

Comment:
@cj-f you'll have to do Dict (capital D) `Optional[Dict[str, Sequence[str]]]` because this needs to support python 3.8+. That's why the older python tests are failing

Reply:
ya I noticed, pushed that updated